src/main/java/androidx/view/View.kt

```diff
@@ -171,3 +171,36 @@ fun View.toBitmap(config: Bitmap.Config = Bitmap.Config.ARGB_8888): Bitmap {
     }
     return Bitmap.createBitmap(width, height, config).applyCanvas(::draw)
 }
+
+/** Show current view */
```

Comment:
Missing period. [Comments should be sentences.](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).

Reply:
What about 
    
    /** Makes view visible. */

Reply:
@JakeWharton thx for link. I will read the article and try to leave comments better for next commit. Prior to that, simply, did not use comments except in rare cases

Reply:
To be extra clear you could add a sentence saying it's equivalent to calling `view.visibility = View.VISIBLE`.